apps/run.go

```diff
@@ -371,8 +372,11 @@ func (runCtx *runContext) connectToRunServer(ctx context.Context) (*http.Respons
 		return nil, nil, errgo.Mask(err, errgo.Any)
 	}
 
-	connection, _ := conn.Hijack()
-	return res, connection, nil
+	if resp.StatusCode != http.StatusOK {
+		return resp, nil, nil
+	}
```

Comment:
suggestion: I think it should return an error in this case, what do you think?

Reply:
I wasn't 100% sure about that part, because I've encountered some 202 while testing on staging, and 202 is not 200 (yet).

What could be the best solution ? add a retry and timeout if 2O2 and for everything else that is not 200 or 202 return an error ?